### DISCUSSION TITLE
In Foundry Develpment Lesson 3: Finishing the setup add new path of AggregatorV3Interface

### DISCUSSION THREAD

**Discussion by DappCoderr** [Cyfrin/foundry-full-course-cu#1812] [Show and tell]
Don't use `forge install smartcontractkit/chainlink-brownie-contracts@1.1.1 --no-commit` (https://github.com/smartcontractkit/chainlink-brownie-contracts); this will throw an error. If you don't get an error, you are good to go. 👍

Otherwise try to use this new path forge `install smartcontractkit/chainlink@v2.12.0 --no-commit` (https://github.com/smartcontractkit/chainlink/)

**Comment by EngrPips** [comment]
Thanks for the heads-up @DappCoderr.

**Comment by danstoyanov** [comment]
It works now with the ` forge install smartcontractkit/chainlink@v2.12.0 --no-commit` , Thanks!🙏

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
